src/pages/tasks/TaskAssigneeSelectorModal.tsx

```diff
@@ -76,7 +67,11 @@ function useOptions({reports}) {
             true,
         );
 
-        const headerMessage = OptionsListUtils.getHeaderMessage(recentReports?.length + personalDetails?.length !== 0 || currentUserOption, Boolean(userToInvite), debouncedSearchValue);
+        const headerMessage = OptionsListUtils.getHeaderMessage(
+            recentReports?.length + personalDetails?.length !== 0 || Boolean(currentUserOption),
```

Comment:
```suggestion
            recentReports?.length + personalDetails?.length !== 0 || !!currentUserOption,
```

Reply:
I think Boolean is better since it is straight forward conversion